Add updated publish documentation section (#121)
bump-sh/docs: src/_help/publish-documentation/documentation-access-management.md

---
title: Documentation access management
---

- TOC
{:toc}

## Public or Private?

> Bump.sh allows you to choose whether a documentation is public or private.

Reviewer: We use "a documentation" here, and then just "documentation"

By "public," we mean documentation that is accessible to any visitor who has access to it, either by finding it through a direct link or through a search engine.
Public documentation is indexed by search engines and can be found by anyone without any restrictions. It is not possible to restrict access to any part of your documentation; it will be fully accessible.

Private documentation, on the other hand, is only accessible to users who have been individually invited. There are two types of invited users: those who are part of the same organization and "external" guests.

> When documentation is set as private, it is completely private. It is excluded from indexing by search engines, and only authenticated users who know its URL can access it.

Reviewer 2: ```suggestion
Private docs require user authentication to be accessible. They are not indexed by search engines.
```

Author: 👍

![](/images/help/documentation-access-choice.png)

> ## Private

Reviewer: I think we should better start with the members of the organization, and then "External guests".

### Invite external guests

> To invite people to access a private documentation, click on the `Share Access` button, which is visible in the settings of that documentation.

Reviewer: People from outside of your organization.

![](/images/help/share-access-button.png)

From there, you just need to enter the email address of the invited person.

![](/images/help/share-access.png)

> They will receive an email inviting them to sign up on Bump.sh. This step is essential to ensure the security of access to your documentation.

Reviewer: Should we add a note about the fact that this isn't possible (yet) on hubs?

Author: Not 100% sure, I think that should be in Hubs section instead?

### Invite members of an organization

Inviting a colleague or teammate to become a member of [your organization](/help/organizations/) can be done from the settings of that organization. Simply enter the email address of the person in question.

> ![](/images/help/org-add-member.png)

Reviewer: It's not about "Inviting", I think this part is just "Members of the organization", and the content should probably just be something like "Any member of your organization can access any doc" or something like that.

## Public

Access to your public documentations is solely determined by its URL and its discoverability by search engines.

> Your public documentations are accessible at the URL `https://bump.sh/your-slug/your-documentation`.

Reviewer 2: A part le pluriel de documentation, je me demande dans quelle mesure cette phrase invite à une note type "pour utiliser un custom domain, allez voir tel bout de doc"?

Reviewer: The URL is wrong. `https://bump.sh/your-slug/doc/your-documentation` (and it can also be under a `/hub`) or a custom domain, yep.

Author: 👍

> > Any changes to your slug/username will result in a change in the URL of your public documentations, without redirection from our side. Make sure to update the URLs you publish or share with your API users accordingly.

Reviewer: Or better: use a custom domain ;)

---
title: Documentation access management
---

- TOC
{:toc}

## Public or Private?

Bump.sh allows you to choose whether documentation is public or private.

By "public," we mean documentation that is accessible to any visitor who has access to it, either by finding it through a direct link or through a search engine.
Public documentation is indexed by search engines and can be found by anyone without any restrictions. It is not possible to restrict access to any part of your documentation; it will be fully accessible.

Private documentation, on the other hand, is only accessible to users who have been individually invited. There are two types of invited users: those who are part of the same organization and "external" guests.
They require user authentication to be accessible and are not indexed by search engines.

![](/images/help/documentation-access-choice.png)

## Private

### Members of an organization

Members of an organization have access to all the documentation contained within that organization.

> Inviting a colleague or teammate to become a member of [your organization](/help/organizations/) can be done from the settings of that organization. Simply enter the email address of the person in question.

Reviewer 3: ```suggestion
Inviting a colleague or teammate to become a member of [your organization](/help/organizations/) can be done from the settings of that organization by simply entering their email address in the dedicated form.
```
